mod/provider/cloudinary.js

```diff
@@ -1,32 +1,71 @@
-const cloudinary_v2 = require('cloudinary').v2
+const { createHash } = require('crypto');
+
+// 1: api_key
+// 2: api_secret
+// 3: cloud_name
+const cloudinary = process.env.CLOUDINARY_URL?.replaceAll("://", " ").replaceAll(":", " ").replaceAll("@", " ").split(" ")
```

Comment:
I am going to fix the double quotes in the cloudinary provider module. As this strays away from our common code standards.